tools/restart-services

```diff
@@ -1,10 +1,13 @@
 #!/bin/bash
 
+sudo service gearman-job-server stop
```

Comment:
Should we stop gearman before we stop the archivematica services that depend on it?